.github/workflows/stackhpc-all-in-one.yml

```diff
@@ -70,14 +70,18 @@ jobs:
   all-in-one:
     name: All in one
     if: inputs.if
-    runs-on: [self-hosted, stackhpc-kayobe-config-aio]
+    runs-on: arc-skc-aio-runner
     permissions: {}
     env:
       KAYOBE_ENVIRONMENT: ci-aio
       KAYOBE_VAULT_PASSWORD: ${{ secrets.KAYOBE_VAULT_PASSWORD }}
       KAYOBE_IMAGE: ${{ inputs.kayobe_image }}
     steps:
-      - uses: actions/checkout@v3
+      - name: Install or Update Git.
+        run: |
+          sudo apt-get update && sudo apt-get install -y git && sudo apt-get install -y nodejs && sudo apt-get install -y unzip && sudo apt-get install -y nodejs && sudo apt-get install -y -f && sudo apt-get build-dep build-essential
```

Comment:
Perhaps try this action rather than installing everything manually: https://github.com/marketplace/actions/install-package

Reply:
Tried it but it doenst use the correct package manager and would have run apt-get manually regardless, they even suggest not using their action for this case at the bottom of their document.

Reply:
Fixed their action not being able to find the packages.